raiden/transfer/channel.py

```diff
@@ -262,6 +297,97 @@ def is_secret_known_onchain(end_state: NettingChannelEndState, secrethash: Secre
     return secrethash in end_state.secrethashes_to_onchain_unlockedlocks
 
 
+def is_valid_channel_total_withdraw(channel_total_withdraw) -> bool:
+    """Sanity check for the channel's total withdraw.
+
+    The channel's total deposit is:
+
+        p1.total_deposit + p2.total_deposit
+
+    The channel's total withdraw is:
+
+        p1.total_withdraw + p2.total_withdraw
+
+    The smart contract forces:
+
+        - The channel's total deposit to fit in a UINT256.
+        - The channel's withdraw must be in the range [0,channel_total_deposit].
+
+    Because the `total_withdraw` must be in the range [0,channel_deposit], and
+    the maximum value for channel_deposit is UINT256, the overflow bellow must
+    never happen, otherwise there is a smart contract bug.
+    """
+    return channel_total_withdraw <= UINT256_MAX
+
+
+def is_valid_withdraw_request_signature(
```

Comment:
The next three functions are (apart from parameter names and types) completely identical. That seems like a nice source for future bugs.

Reply:
We plan on a follow up PR to address some of the nitpicks we noticed here and there.

I think @hackaugusto had a goal when he did this. Was it because we can change the signature parameter for each of those messages?